sdk/core/azure-core/azure/core/pipeline/policies/_universal.py

```diff
@@ -205,6 +206,11 @@ def __init__(self, base_user_agent=None, **kwargs):  # pylint: disable=super-ini
             )
 
         if application_id:
+            if application_id.length > self._MAX_APPLICATION_ID_LENGTH:
+                raise ValueError("'applicationId' length cannot be greater than " + self._MAX_APPLICATION_ID_LENGTH)
```

Comment:
If the app_id is too long, I prefer truncating the string rather than failing the app. 

Your opinions? @johanste  @annatisch

Reply:
It throws error in C# and java. Here are the similar validation 
https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/core/azure-core/src/main/java/com/azure/core/util/ClientOptions.java#L57
https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/src/DiagnosticsOptions.cs#L109